examples/kind-quickstart/init-job.yaml

```diff
@@ -7,7 +7,7 @@ spec:
     spec:
       containers:
         - name: feast-apply
-          image: feastdev/feature-server:0.40.1
+          image: quay.io/feastdev/feature-server:0.40.1
```

Comment:
are we migrating old version as well ? it should be from o.46.0 onward right ? 

Reply:
that's a quickstart thing... i can't presume to increase the version. it may break that feast quickstart. imo, that would be a separate PR

Reply:
that image won't exist in docker.io though, so i guess we can change the version

Reply:
Yup that's my point, we can change it to `latest` tag ?

Reply:
fixed